recipes/rake-nltk/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "rake_nltk" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "67759e32ef9be82bbfc78efdb92db56fc43bc7c98053dcd415f6ddf576b4ea62" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - nltk
```

Comment:
Is nltk really a build-time dependency or just runtime?

Reply:
nltk is only a runtime dependency. I removed it from the meta.yaml file.

Reply:
@cu8blank, 
Please add nltk_data as a build-time and runtime dependency. This should fix the download issue as explained by @jakirkham below.

Once that is fixed the only other thing I can think about is to check if rake can be bundled as a noarch package..